src/app/content/components/utils/attachHighlight.ts

```diff
@@ -0,0 +1,17 @@
+import Highlighter, { Highlight, SerializedHighlight } from '@openstax/highlighter';
+import Sentry from '../../../../helpers/Sentry';
+
+const attachHighlight = (
+  highlight: Highlight | SerializedHighlight,
+  highlighter: Highlighter
+) => {
+  highlighter.highlight(highlight);
+  const hl = highlight instanceof SerializedHighlight
+    ? highlighter.getHighlight(highlight.id)
+    : highlight;
+  if (!hl || !hl.isAttached()) {
+    Sentry.captureException(`Highlight with id: ${highlight.id} has not been attached.`);
```

Comment:
captureException takes an `Error object`, i think you want `Sentry.error()` which takes a string